src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -2794,4 +2794,14 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         }
     }
 
+    void TermControl::OwningHwnd(uint64_t owner)

```

Comment:
Nit: vaguely prefer the `HostingWindow` parlance; we can take that as a followup (look at TSE's `IHostedInWindow`)